cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1864,7 +1864,7 @@ endif()
 
 #---Check for ZeroMQ when building RooFit::MultiProcess--------------------------------------------
 
-if (roofit_multiprocess AND NOT MSVC)
+if (roofit_multiprocess AND NOT WIN32)
```

Comment:
Rather than doing this here, maybe it's better to have an `if (roofit_multiprocess AND WIN32)` in `RootBuildOptions.cmake` that errors out saying it's not supported on Windows (similar to other options that have such checks). Then we have a single place to control this, rather than checking in every `if (root_multiprocess)` if we are on Windows or not.

Reply:
Good idea. I have pushed two new commits (I will fixup rebase them into the proper commits later, after CI shows it works) that implement this.